src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -929,6 +929,37 @@ public void withSizeOperatorNoMatch() {
           .body("data.documents", hasSize(0));
     }
 
+    @Test
+    public void withSizeOperatorNotZero() {
+      String json =
+          """
+                  {
+                      "find": {
+                          "filter": {
+                              "$not": {
+                                  "tags": {
+                                      "$size": 0
+                                  }
+                              }
+                          }
+                      }
+                  }
+                      """;
+
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("status", is(nullValue()))
+          .body("errors", is(nullValue()))
+          // should have all the documents
+          .body("data.documents", hasSize(6));
```

Comment:
For completeness, would be great to have a document with `tags: []` and test that's excluded from the output